bin/yii

```diff
@@ -37,12 +42,23 @@ use Yiisoft\Yii\Console\Output\ConsoleBufferedOutput;
         fwrite(STDERR, $message);
         exit(1);
     }
+
     require_once $autoloadPath;
 
-    $params = require Builder::path('params');
+    // Don't do it in production, assembling takes it's time
+    if (getenv('env') === 'dev') {
+        Builder::rebuild();
+    }
+
+    $startTime = microtime(true);
+
+    $container = new Container(
```

Comment:
It makes it impossible to replace container with another implementation.

Reply:
It's yii-console, so it makes sense to me to use the yii-di container.